packages/fluentui/docs/src/examples/components/Datepicker/Usage/DatepickerFormatExample.shorthand.tsx

```diff
@@ -0,0 +1,13 @@
+import { Datepicker } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const DatepickerFormatExample = () => {
+  const customFormat = (date: Date) => {
```

Comment:
```suggestion
  const customFormatter: SOME_TYPE = (date) => {
```

Does thing has typings? Can we use them?